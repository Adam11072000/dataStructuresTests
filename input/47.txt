init 182
mergeSuperPixels 82 174 121
addImage 11
setLabelScore 11 48 43 11
setLabelScore 11 120 83 166
addImage 11
resetLabelScore 11 141 0
setLabelScore 196 98 -3 215
quit