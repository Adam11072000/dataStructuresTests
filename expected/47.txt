init done.
mergeSuperPixels: FAILURE
addImage: SUCCESS
setLabelScore: SUCCESS
setLabelScore: SUCCESS
addImage: FAILURE
resetLabelScore: INVALID_INPUT
setLabelScore: INVALID_INPUT
quit done.
